intranet/static/js/login.js

```diff
@@ -1,49 +1,61 @@
 /* global $ */
-$(document).ready(function() {
+$(function() {
     /* Input */
-    var $username = $("input[name=username]");
-    var $password = $("input[name=password]");
-    if(!$username.hasClass("error") && $password.hasClass("error")) {
+    var $username = $("input[name=username]"),
+        $password = $("input[name=password]");
+
+    if (!$username.hasClass("error") && $password.hasClass("error")) {
         $password.focus();
     } else {
         $username.focus();
     }
 
     $(".logo").click(function() {
-        location.href = (window.osearch ? "/?"+window.osearch.substring(0, window.osearch.length-1) : "/");
+        location.href = (window.osearch ? "/?" + window.osearch.substring(0, window.osearch.length - 1) : "/");
     });
 
     $(".git-version").click(function(e) {
         location.href = $(this).attr("data-github-url");
     });
 
     $("input[type=submit]").click(function(e) {
-        if(typeof runEgg == 'function' && runEgg($("#id_username").val())) { e.preventDefault(); return; }
+        if (typeof runEgg === 'function' && runEgg($("#id_username").val())) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["runEgg" is not defined.](https://www.codacy.com/app/tjhsst-ion/ion/file/2743306441/issues/source?bid=3212528&fileBranchId=3355681#l22)